src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -43,12 +48,25 @@ public LogicManager(Model model, Storage storage) {
         addressBookParser = new AddressBookParser();
     }
 
+
+    @Override
+    public void validateCommand(String commandText, boolean showingMatchAutoView)
+            throws ParseException, CommandException {
```

Comment:
Goto CommandException, and there will be an error in the documentation, it seems to be an import error.